__tests__/app.test.js

```diff
@@ -38,7 +38,7 @@ describe("app", () => {
     });
   });
   describe("GET /api/articles/:article_id", () => {
-    test("status 200: returns an object containing article information", () => {
+    test.only("status 200: returns an object containing article information", () => {
```

Comment:
get rid of your `.only`s before you push up

Reply:
✅